core/templates/dev/head/footer.html

```diff
@@ -21,13 +19,15 @@ <h4 translate="I18N_FOOTER_TEACH_LEARN_ALL_CAPS"></h4>
           <li><a href="/teach#teach" translate="I18N_FOOTER_TEACH"></a></li>
           <li><a href="/teach#playbook" translate="I18N_FOOTER_PARTICIPATION_PLAYBOOK"></a></li>
           <li><a href="/library" translate="I18N_FOOTER_BROWSE_LIBRARY"></a></li>
+          <li><a href="/forum" translate="I18N_FOOTER_FORUM"></a></li>
         </ul>
       </div>
       <div class="col-sm-3">
-        <h4 translate="I18N_FOOTER_CONTACT_ALL_CAPS"></h4>
```

Comment:
Optional: Is this old translation used anywhere else? If not, may want to remove it from the i18n json files.